### PR TITLE
refactor metrics names used in client

### PR DESCRIPTION
**Pull request by micossow**

### CODE REVIEW

**Review by MichalStopyra — APPROVED** [review]

File: clients/core/src/main/scala/mqperf/Metrics.scala [hunk 1]
```diff
@@ -7,7 +7,7 @@ object Metrics {
 
   object Sender {
     val messageCounter: Counter =
-      Counter.build("mqperf_sent_total", "number of sent messages").labelNames(testIdLabelName).register()
+      Counter.build("mqperf_sent", "number of sent messages").labelNames(testIdLabelName).register()
```
Comment (lukaszlenart) [inline clients/core/src/main/scala/mqperf/Metrics.scala]: This represent all the sent messages during one test session? one execution? Wouldn't be a good idea to extend the description about this metric? And same above with `mqperf_received`?

Comment (micossow) [reply clients/core/src/main/scala/mqperf/Metrics.scala]: actually it's a simple http server app exposing metrics. It's used as a part of a test, but it's not the test itself, so terms like test session/test execution don't really apply here.

Comment (lukaszlenart) [reply clients/core/src/main/scala/mqperf/Metrics.scala]: aha, so `total` was misleading here, got it!

**Review by lukaszlenart — APPROVED** [review]
LGTM 👍

**Pull request merged**